policy-controller/k8s/index/src/index.rs

```diff
@@ -707,15 +707,19 @@ impl kubert::index::IndexNamespacedResource<k8s_gateway_api::HttpRoute> for Inde
 
     fn reset(
         &mut self,
-        routes: Vec<k8s_gateway_api::HttpRoute>,
+        routes: Vec<k8s::policy::HttpRoute>,
         deleted: HashMap<String, HashSet<String>>,
     ) {
         self.reset_route(routes, deleted)
     }
 }
 
-impl kubert::index::IndexNamespacedResource<k8s::policy::HttpRoute> for Index {
-    fn apply(&mut self, route: k8s::policy::HttpRoute) {
+// NOTE: This implementation is currently unused, as the policy controller only
+// supports indexing the `policy.linkerd.io` version of the `HTTPRoute` CRD.
+// This implementation will be used if support for the
+// `gateway.networking.k8s.io` CRD is added again in the future.
+impl kubert::index::IndexNamespacedResource<k8s_gateway_api::HttpRoute> for Index {
```

Comment:
FWIW, I think it would be fine to remove this indexer, since it's actually dead code. I was just hoping to avoid larger changes to code that is otherwise functional.

Reply:
okay, can do